tests/upload_test.go

```diff
@@ -891,6 +864,71 @@ var _ = Describe("[rfe_id:138][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		Expect(deleted).To(BeTrue())
 	})
 
+	It("Upload an image exactly the same size as DV request (bz#2064936)", func() {
+		// This image size and filesystem overhead combination was experimentally determined
+		// to reproduce bz#2064936 in CI when using ceph/rbd with a Filesystem mode PV since
+		// the filesystem capacity will be constrained by the PVC request size.
+		size := "858993459"
+		fsOverhead := "0.055" // The default value
+		tests.SetFilesystemOverhead(f, fsOverhead, fsOverhead)
+
+		volumeMode := v1.PersistentVolumeMode(v1.PersistentVolumeFilesystem)
+		accessModes := []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce}
+		dvName := "upload-dv"
+		By(fmt.Sprintf("Creating new datavolume %s", dvName))
+		dv := utils.NewDataVolumeForUploadWithStorageAPI(dvName, size)
+		dv.Spec.Storage.AccessModes = accessModes
+		dv.Spec.Storage.VolumeMode = &volumeMode
+		dataVolume, err = utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dv)
+		pvc = utils.PersistentVolumeClaimFromDataVolume(dataVolume)
+
+		By("verifying pvc was created, force bind if needed")
+		pvc, err := utils.WaitForPVC(f.K8sClient, pvc.Namespace, pvc.Name)
+		Expect(err).ToNot(HaveOccurred())
+		f.ForceBindIfWaitForFirstConsumer(pvc)
+
+		phase := cdiv1.UploadReady
+		By(fmt.Sprintf("Waiting for datavolume to match phase %s", string(phase)))
+		err = utils.WaitForDataVolumePhase(f.CdiClient, f.Namespace.Name, phase, dataVolume.Name)
+		if err != nil {
+			dv, dverr := f.CdiClient.CdiV1beta1().DataVolumes(f.Namespace.Name).Get(context.TODO(), dataVolume.Name, metav1.GetOptions{})
+			if dverr != nil {
+				Fail(fmt.Sprintf("datavolume %s phase %s", dv.Name, dv.Status.Phase))
+			}
+		}
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Get an upload token")
+		token, err := utils.RequestUploadToken(f.CdiClient, pvc)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(token).ToNot(BeEmpty())
+
+		By("Create empty image with specific size")
```

Comment:
You can remove this including the defer.

Reply:
yes